manifests/p/PulseBroswer/PulseBrowser/1.0.0-a.87/PulseBroswer.PulseBrowser.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: PulseBroswer.PulseBrowser
+PackageVersion: 1.0.0-a.87
+InstallerType: burn
+InstallerSwitches:
+  Custom: /S
+UpgradeBehavior: install
+ReleaseDate: 2023-12-01
+    Dependencies:
+      PackageDependencies:
+        - PackageIdentifier: Microsoft.VCRedist.2015+.x64

```

```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

I'm not sure where the extra spaces came from.